worldengine/cli/main.py

```diff
@@ -395,9 +394,12 @@ def main():
 
     if args.seed is not None:
         seed = int(args.seed)
+        if not 0 <= seed <= 65535:  # platec might not be able to handle integers larger than uint16; TODO: check this
+            print("Seed has to be in the range of 0 and 65535.")
```

Comment:
Ok, I'll change it.

Since you're already here: Could you elaborate on the seed-limit for platec a bit? Is there really a uint16 in use?